src/main/kotlin/net/atos/zac/zaken/ZakenService.kt

```diff
@@ -68,13 +70,21 @@ class ZakenService @Inject constructor(
                         explanation
                     )
                 }
-                indexeerService.indexeerDirect(
-                    zaak.uuid.toString(),
-                    ZoekObjectType.ZAAK,
-                    false
-                )
+                // unless OpenNotificaties is disabled, we don't need to do any indexing here,
+                // because we will do it based on notifications from OpenZaak
+                if (configuratieService.notificationsDisabled) {
```

Comment:
As said before I would turn this around. `if (!configuratieService.notificationsEnabled)`.. much easier to get your head around I think.

Reply:
Will remove